docs/TEAM.md

```diff
@@ -34,6 +34,7 @@
 | |[Aravind Sivasailam](https://github.com/aravindsivasailam) | I'm a product engineer living in US with 2 years of experience in software testing and project management. New to software development and wish to improve in HTML, CSS and JS. |
 | |[Edwin Mancipe](https://github.com/efmg0325)|I am from Colombia. I have a bit of knowledge about HTML, CSS, Javascript, Python and SQL. I love to listeninng to Heavy Metal music. I like second life web browser games.|
 | |[Edori Atiri](https://github.com/EdoriAtiri) | From Nigeria. Began to learn Web Development this year. I hope to learn and contribute while I'm here.|
+| |[OmoloG] (https://github.com/geowen74) | Fom Nairobi, Kenya. Have Knowledge on HTML, CSS and some basic on javascrip and Mysql. Aspiring to be a full-stack developer. I love to learn how to code.
```

Comment:
`[OmoloG] (https://github.com/geowen74)` should be `[OmoloG](https://github.com/geowen74)`. Remove extra space between both. Verified once in the preview mode.

Reply:
extra space where